### DISCUSSION TITLE
Why is it not possible to change the website settings?

### DISCUSSION THREAD

**Discussion by tikita** [datenstrom/community#703] [Ask a question] [answered]
Almost funny ;-) when I want to change the settings: in one of three installations, the website details can not be changed, or can not be accessed, the link is not present.

See the attached picture.

All three installations run at the same provider with Yellow 0.8.18, PHP 7.4.25.

Does anyone know where I might need to do some tweaking?

Greetings, Heiner

![yellow-merkwuerdigkeit](https://user-images.githubusercontent.com/6100237/141692615-3e260da3-2856-4f55-9fb8-47f45da47a3c.png)

_Originaltext auf deutsch:

Fast schon lustig ;-) wenn ich die Einstellungen ändern möchte: bei einer von drei Installationen können die Angaben zur Webseite nicht geändert, bzw. nicht aufgerufen werden, der Link ist nicht vorhanden.

Siehe dazu das anhängende Bild.

Alle drei Installationen laufen beim selben Provider mit Yellow 0.8.18, PHP 7.4.25.

Weiß jemand, wo ich da vielleicht etwas nachbessern muss?_

**Comment by markseuffert** [comment, marked as answer]
You probably don't have the necessary user access rights to configure the website. This is easy to change. :smile:

Open file `system/extensions/yellow-user.ini`, search for your user account, check the `Access` setting and add `configure`. This will be done automatically for the administrator of a new website, otherwise correct your user account. The following user access rights are supported in Datenstrom Yellow 0.8.18:

`create` = user can create page  
`edit` = user can edit page  
`delete` = user can delete page  
`restore` = user can restore deleted page  
`upload` = user can upload media files  
`configure` = user can configure website  
`update` = user can update website  

See also https://github.com/annaesvensson/yellow-edit

**Comment by tikita** [comment]
Thank you Mark for your quick reply!

Yes it was, manually corrected the line in `system/extensions/yellow-user.ini`:

**Access: create, edit, delete, restore, upload, configure, update**

Greetings, Heiner
